client/src/components/overview/GalleryView.jsx

```diff
@@ -33,6 +33,20 @@ const StyledOverviewGrid = styled.div`
 
 function Overview(props) {
   const [count, setCount] = useState(0);
+  useEffect(() => {
+    axios({
+      method: 'get',
+      url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/37314/styles',
```

Comment:
Perhaps set the product id as a variable within the url so that it will be easier to replace later on.

Reply:
Good idea, I will make a trello ticket for that now.